### PR TITLE
benches: bench mvcc reader/txn

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**

### CODE REVIEW

**Comment by ngaut** [comment]
Any bench results?

File: benches/mvcc/bench_mvcc.rs [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    let engine = engine::new_local_engine(path.path().to_str().unwrap(), ALL_CFS).unwrap();
+    init_data(engine.as_ref(), 10 * MB, true);
+
+    let mut statistics = Statistics::default();
+    let ctx = Context::new();
+    let snapshot = engine.snapshot(&ctx).unwrap();
+    let mut reader = MvccReader::new(snapshot.as_ref(), &mut statistics, None, true, None);
+
+    b.iter(|| {
+        reader.get(&make_key(b"key_a_1000"), 1001);
```
Comment (siddontang) [inline benches/mvcc/bench_mvcc.rs]: need to use a random key here?

File: benches/mvcc/bench_mvcc.rs [hunk 2]
```diff
@@ -0,0 +1,212 @@
+
```
Comment (BusyJay) [inline benches/mvcc/bench_mvcc.rs]: Missing license.

**Comment by siddontang** [comment]
Any update?

**Pull request closed without merging**